README.md

```diff
@@ -1,2 +0,0 @@
-# MinigolfWithFriends
```

Comment:
These have been causing me way too much pain. Better to delete both and then add a single one later if need be.